test/e2e/preset_test.go

```diff
@@ -56,6 +56,7 @@ func loadTestEnvVars() {
 	aiModelsRegistry = utils.GetEnv("AI_MODELS_REGISTRY")
 	aiModelsRegistrySecret = utils.GetEnv("AI_MODELS_REGISTRY_SECRET")
 	supportedModelsYamlPath = utils.GetEnv("SUPPORTED_MODELS_YAML_PATH")
+	azureClusterName = utils.GetEnv("AZURE_CLUSTER_NAME")
```

Comment:
Is this used anywhere?

Reply:
Yes this is used by fine tuning e2e - outputRegistryUrl. e2e for fine tuning previously was uploading to local ACR now we are using temp ACR